concordia/templates/transcriptions/campaign_list.html

```diff
@@ -13,28 +13,24 @@
 {% endblock breadcrumbs %}
 
 {% block main_content %}
-<div class="container-fluid py-3">
-    <div class="row">
-        <div class="d-flex flex-wrap col-12 mb-3 justify-content-around">
-            {% for campaign in campaigns %}
-                <div class="bg-lightest-gray w-100 mb-3">
-                    <div class="row">
-                        <div class="col-12 col-md-8 px-3">
-                            <a href="{% url 'transcriptions:campaign-detail' campaign.slug %}">
-                                <h1 class="campaign-title">{{ campaign.title }}</h1>
-                            </a>
-                            {{ campaign.short_description|linebreaksbr }}
-                        </div>
-                        <div class="col-12 col-md-4">
-                            <a href="{% url 'transcriptions:campaign-detail' campaign.slug %}">
-                            <img src="{{ MEDIA_URL }}{{ campaign.thumbnail_image }}" class="img-fluid" alt="{{ campaign.title }} image">
-                            </a>
-                        </div>
-                    </div>
+<div class="container py-3">
+    <h1 class="mb-5 sr-only">Campaigns</h1>
+    <ul class="list-unstyled">
+        {% for campaign in campaigns %}
+        <li class="p-4 mb-1 bg-footer">
```

Comment:
This block should be indented to show the loop structure

Reply:
This is fixed and committed.